add b = 0 field calrfication, and define truncated division for clarity (#207)
AleoHQ/welcome: documentation/leo/04_operators.md

```
#### Description

Performs division of the first operand by the second, storing the result in the destination. The operation halts if division by zero is attempted.

For integer types, this operation performs truncated division. Truncated division rounds towards zero, regardless of the sign of the operands. This means it cuts off any digits after the decimal, leaving the largest whole number less than or equal to the result.

For example:

1. `7 / 3` yields `2`, not `2.3333`.
2. `-7 / 3` yields `-2`, not `-2.3333`.

> A constraint is added to check for underflow. Underflow occurs when dividing the minimum value of a signed integer type by -1. For example, `-128i8 / -1i8` would result in underflow, since 128 cannot be represented as an `i8`.

Reviewer: ```suggestion
The operation halts if there is an underflow. Underflow occurs when dividing the minimum value of a signed integer type by -1. For example, `-128i8 / -1i8` would result in underflow, since 128 cannot be represented as an `i8`.
```
